index.js

```diff
@@ -16,12 +16,12 @@ module.exports = function (hex, options = {}) {
 	let alpha = 255;
 
 	if (hex.length === 8) {
-		alpha = parseInt(hex.slice(6, 8), 16);
+		alpha = parseFloat((parseInt(hex.slice(6, 8), 16) / 255).toFixed(4));
```

Comment:
Why are you limiting it to 4 fractional digits? I don't really see the benefit except for presentation. This needlessly limits the accuracy.

Reply:
I honestly have no idea why I've done that, mistake on my end, sorry about that. 😄